MenuApplets/Clock/main.cpp

```diff
@@ -227,21 +254,30 @@ class ClockWidget final : public GUI::Widget {
 
     virtual void paint_event(GUI::PaintEvent& event) override
     {
-        auto time_text = Core::DateTime::now().to_string();
+        auto current_time = Core::DateTime::now();
+        auto time_text = current_time.to_string();
         GUI::Painter painter(*this);
         painter.fill_rect(event.rect(), palette().window());
         painter.draw_text(event.rect(), time_text, Gfx::Font::default_font(), Gfx::TextAlignment::Center, palette().window_text());
+
+        if (m_analog_clock_widget)
```

Comment:
Can `m_analog_clock_widget` be empty? Line 86 seems pretty unconditional to me. If it can't be empty, then the `if` seems silly :P

Reply:
Oops. My bad